modules/4_nodes/nodes.tf

```diff
@@ -23,18 +23,46 @@ resource "random_id" "label" {
     byte_length = "2"
 }
 
+data "ignition_systemd_unit" "restart-NetworkManager" {
+    name        = "NetworkManager.service"
+    dropin {
+        name    = "restart.conf"
+        content = <<EOF
+[Service]
+Restart=always
+RuntimeMaxSec=300
+EOF
+    }
+}
+
+data "ignition_file" "dhcp-timeout" {
+    overwrite   = true
+    mode        = "755" // 0644
+    path        = "/etc/NetworkManager/conf.d/99-dhcp-timeout.conf"
+    content {
+        content = <<EOF
+[connection]
+ipv4.dhcp-timeout=2147483647
```

Comment:
We should have a good compromise between fast and slow failover. This will endlessly wait in a buggy network and can create more issues than it resolve. Is there a decent default ?

Reply:
I have tried 120 which did not help. From 45 to 120 is a big jump already.
I will do some more tests wrt this and will update it later if necessary.